scripts/test.js

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env node
+
+process.env.BABEL_ENV = 'test';
+process.env.NODE_ENV = 'test';
+
+// Load the create-react-app config.
+const jestConfig = require('react-scripts/scripts/utils/createJestConfig');
+
+// Map React's testing classes to Preact
+const newJestConfig = function() {
+	const config = jestConfig.apply(this, arguments);
+	config.moduleNameMapper = Object.assign({}, config.moduleNameMapper, {
```

Comment:
whenever `jest` tries to load or mock any `react*` modules, it will get the `preact*` equivalents.